From e2b3be80e2c9ecb3a9fa1a9448621b1190b21823 Mon Sep 17 00:00:00 2001
From: John Levon <john.levon@joyent.com>
Date: Mon, 2 Sep 2019 15:41:14 +0000
Subject: [PATCH] OS-7977 -fno-shrink-wrap shouldn't be specified for gcc4
 build Reviewed by: Tim Foster <tim.foster@joyent.com> Approved by: Tim Foster
 <tim.foster@joyent.com>

---
 tun/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/tun/Makefile b/tun/Makefile
index 4941c32..47f7953 100644
--- a/tun/Makefile
+++ b/tun/Makefile
@@ -25,7 +25,9 @@ VER = tun-1.3
 
 include ../Makefile.defs
 
+ifneq ($(PRIMARY_COMPILER_VER),4)
 CFLAGS.64 += -fno-shrink-wrap
+endif
 
 BUILD32 =
 BUILD64 = yes
-- 
2.21.0

